src/lumigo_tracer/wrappers/redis/redis_wrapper.py

```diff
@@ -0,0 +1,85 @@
+import importlib
+from typing import Optional, Dict, List, Union
+
+import time
+import uuid
+
+from lumigo_tracer.libs.wrapt import wrap_function_wrapper
+from lumigo_tracer.lumigo_utils import lumigo_safe_execute, get_logger, lumigo_dumps
+from lumigo_tracer.spans_container import SpansContainer
+
+
+REDIS_SPAN = "redis"
+
+
+def command_started(
+    command: str, request_args: Union[Dict, List[Dict]], connection_options: Optional[Dict]
+):
+    span_id = str(uuid.uuid4())
+    host = (connection_options or {}).get("host")
+    port = (connection_options or {}).get("port")
+    SpansContainer.get_span().add_span(
+        {
+            "id": span_id,
+            "type": REDIS_SPAN,
+            "started": int(time.time() * 1000),
```

Comment:
Maybe extract `* 1000` to a function so it will be clear what to do here (convert miliseconds and secodns)